app/frontend/src/router/minesOperatorScreening.js

```diff
@@ -1,77 +1,82 @@
 /**
  * Mines Operator Screening Form Routes
  */
+import { FormNames } from '@/utils/constants';
+
+const baseName = 'MinesOperatorScreening';
+const baseTitle = 'Industrial Camps';
+
 export default [
   {
-    path: '/minesoperatorscreening',
+    path: `/${FormNames.MINESOPERATORSCREENING}`,
     component: () => import(/* webpackChunkName: "minesoperatorscreening" */ '@/views/MinesOperatorScreening.vue'),
     children: [
       {
         path: '',
-        name: 'MinesOperatorScreeningForm',
+        name: `${baseName}Form`,
         component: () => import(/* webpackChunkName: "minesoperatorscreening-form" */ '@/views/minesoperatorscreening/Root.vue'),
         meta: {
-          title: 'Industrial Camps'
+          title: baseTitle
         }
       },
       {
         path: 'admin',
-        name: 'MinesOperatorScreeningAdmin',
+        name: `${baseName}Admin`,
         component: () => import(/* webpackChunkName: "minesoperatorscreening-admin" */ '@/views/minesoperatorscreening/Admin.vue'),
         meta: {
           hasLogin: true,
           requiresAuth: true,
-          title: 'Industrial Camps Admin'
+          title: `${baseTitle} Admin`
         }
       },
       {
         path: 'admin/dashboard',
-        name: 'MinesOperatorScreeningDashboards',
+        name: `${baseName}Dashboards`,
         component: () => import(/* webpackChunkName: "minesoperatorscreening-dashboard" */ '@/views/minesoperatorscreening/Dashboards.vue'),
         meta: {
           hasLogin: true,
           requiresAuth: true,
-          title: 'Industrial Camps Admin'
+          title: `${baseTitle} Admin`
```

Comment:
${baseTitle} Dashboards?

Reply:
I don't think it was a copy-paste error - we had it originally as just an admin panel - easy to change but was refactoring this to maintain existing title consistency with prod.